tests/queries/0_stateless/03156_groupConcat.sql

```diff
@@ -0,0 +1,21 @@
+DROP TABLE IF EXISTS test_groupConcat;
```

Comment:
also, please, create a test when you 
- aggregate a large amount of data to check serialization
- test exceptions that you implemented in the code